synce_ql_manager.c

```diff
@@ -0,0 +1,125 @@
+/**
+ * @file synce_ql_manager.c
+ * @brief Interface for managing QL for external clock sources
+ * @note SPDX-FileCopyrightText: Copyright 2022 Intel Corporation
+ * @note SPDX-License-Identifier: GPL-2.0+
+ */
+#include <unistd.h>
+#include <pthread.h>
+#include <stdlib.h>
+#include <net/if.h>
+#include "print.h"
+#include "config.h"
+#include "synce_dev.h"
+#include "synce_clock.h"
+#include "synce_ql_manager.h"
+
+void* synce_ql_manager_server_thread(void* arg) {
+	char usage[] = "Usage: SET <dev_name> <sma_name> <QL/EXT_QL> <ql_value>";
+	struct synce_clock *clk = (struct synce_clock*)arg;
+	char ext_src_name[MAX_COMMAND_SIZE];
+	struct sockaddr_in server, client;
+	char response[MAX_RESPONSE_SIZE];
+	char dev_name[MAX_COMMAND_SIZE];
```

Comment:
I cant use that, since I parse the command from  user and I dont want to get to overflow. the assumption is that the user might put longer string than MAX_IFNAME_SIZE

Reply:
changed accrdingly